assets/js/base/components/cart-checkout/totals/coupon/index.tsx

```diff
@@ -95,7 +103,12 @@ export const TotalsCoupon = ( {
 				isLoading={ isLoading }
 				showSpinner={ false }
 			>
-				<div className="wc-block-components-totals-coupon__content">
+				<div
+					className={
```

Comment:
When dealing with changing classes we use the `classnames` package. 
In order to use it you'd have to change the logic you are using. Using string values to detect a state of an element is error-prone (classes usually change, string comparison is tricky), so it's better to work with boolean values. 
I recommend walking away from the `couponFormClass` and introducing `isCouponFormHidden` as a way to check the state of the component. The latter has the advantage of being more explicit about the functionality it handles, it's clear that the form is sometimes hidden and it depends on the value of `isCouponFormHidden`.

Class change should be handled as follows: 

```
import classnames from 'classnames';
...
const formWrapperClass = classnames(
		'wc-block-components-totals-coupon__content',
		{
			'screen-reader-text': isCouponFormHidden,
		}
	);
... 
<div className={formWrapperClass}>
 <form
```

Reply:
This is super helpful. Thank you! I have implemented the suggestion. 👍